modules/atmosphere/shaders/atmosphere_deferred_fs.glsl

```diff
@@ -426,10 +416,16 @@ vec3 inscatterRadiance(inout vec3 x, inout float t, inout float irradianceFactor
     // we can change it on the fly with no precomputations)
     // return radiance * sunRadiance;
     vec3 finalScatteringRadiance = radiance * sunIntensity;
+    
     if (groundHit) {
         return finalScatteringRadiance;
     } else {
-        return ((r-Rg) * invRtMinusRg)*spaceColor.rgb * backgroundConstant + finalScatteringRadiance;
+        //return ((r-Rg) * invRtMinusRg)*spaceColor.rgb + finalScatteringRadiance;
+        //return attenuation * spaceColor.rgb + finalScatteringRadiance;
```

Comment:
`attenuation * spaceColor.rgb + finalScatteringRadiance;` is equivalent to the current return value if attenuation is between 0 and 1. Replace current return with this line?